main.css

```
@@ -0,0 +1,928 @@
+@font-face {
+    font-family: Asap;
+    src: url('./assets/fonts/Asap-Regular.otf')
+}
+@font-face {
+    src: url("./assets/fonts/segoe-wp-light.ttf");
+    font-family: "Segoe WP";
+}
+@font-face {
+    src: url("./assets/fonts/PTS55F.ttf");
+    font-family: "PT Sans";
+}
+@font-face {
+    src: url("./assets/fonts/OpenSans-Regular.ttf");
+    font-family: "Open Sans";
+}
+@font-face {
+    font-family: "PT Sans Narrow";
+    src: url("./assets/fonts/PTN57F.ttf")
+}
+@font-face {
+    font-family: "Lucida Sans";
+    src: url("./assets/fonts/lucida-sans/LSANSI.TTF")
+}
+
+body {
+    width: 1300px; 
+    margin: 0 auto;
+}
+header {    
+    background-image: 
+      url('./assets/images/Layer\ 20.png'),
+      url('./assets/images/croppedL19.png'),
+      url('./assets/images/Layer\ 1.png');
+    background-repeat: no-repeat;
+    height: 566px;
+    text-transform: uppercase; 
+}
+.banner {
+    display: flex;
+    line-height: 85px;
+}
+.logo-banner {   
+    background-image: url("./assets/images/Logo banner.svg");
+    background-repeat: no-repeat;
+    background-position-y: 27px;
+    margin: 0;
+    display: flex;
+    margin-left: 140px;
+    /* margin-top: 28px; */
+    line-height: 85px;
+}
+h1 {
+    margin: 31px 87px;
+    font-family: "Open Sans";
```

use general default for all font-family props
```suggestion
    font-family: "Open Sans", sans-serif;
```